src/app/dashboard/users/Search.tsx

```diff
@@ -0,0 +1,60 @@
+'use client'
+
+import { useDebounce } from '@/hooks'
+import React, { useEffect, useState } from 'react'
+
+import SearchWithButton from '@/components/inputFields/SearchFieldWithButton'
+import SearchWithDebounce, {

```

Comment:
Please follow the consistent name casing in the folder as well as file names.

Reply:
Sure, Thanks